IceNet/polar-modelling-icenet.ipynb

```diff
@@ -0,0 +1,1411 @@
+{
```

Comment:
<p>Maybe I missed it, but is there an explanation somewhere of where the ground truth SIC data comes from?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/PangeoExamples/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>I'm adding the following text to the <strong>Download ground truth SIC </strong>section explaining further details of the the ground truth SIC data.</p><p> </p><blockquote>We additionally download analysis-ready ground truth SIC data from a zenodo repository.</blockquote><blockquote><br></blockquote><blockquote>:::{note}</blockquote><blockquote>The analysis-ready ground truth SIC data were generated by running the script <code>python3 icenet/download_sic_data.py</code> in step **2) Download data** according to the [icenet-paper repository](https://github.com/tom-andersson/icenet-paper). The script downloads and concatenate [OSI-SAF SIC data](https://osisaf-hl.met.no/v2p1-sea-ice-index), OSI-450 (1979-2015) and OSI-430-b (2016-ownards), and saves it as monthly averages in a <code>netCDF</code> file.</blockquote><blockquote>:::</blockquote><p> </p>